kivy/data/style.kv

```diff
@@ -196,7 +196,7 @@
             pos: self._cursor_visual_pos
             size: root.cursor_width, -self._cursor_visual_height
         Color:
-            rgba: self.disabled_foreground_color if self.disabled else (self.hint_text_color if not self.text else self.foreground_color)
+            rgba: (1, 1, 1, 1)
```

Comment:
The previous version had the advantage of automatically adjusting the rendering if any of the `xxx_color` properties, or `disabled` was updated, the new way means lines have to be rerendered, for it to work. so binding on these properties, probably around line 568, of textinput.py, would be necessary, otherwise these color/disabled changes won't be applied until the text (or some other property that is bound) is actually changed.

Reply:
Ah yes, thanks for the help!

Reply:
Does it look fine to you now @tshirtman?